crates/rolldown/tests/rolldown/topics/bundler_esm_cjs_tests/13/artifacts.snap

```diff
@@ -53,8 +53,8 @@ Variant: (format: Cjs)
 const require_foo = require('./foo2.js');
 
 //#region entry.js
-init_foo();
-const foo = (init_foo(), __toCommonJS(foo_exports));
+require_foo.init_foo();
+const foo = (require_foo.init_foo(), require_foo.__toCommonJS(require_foo.foo_exports));
```

Comment:
Here `require_foo.__toCommonJS` is incorrect.

Reply:
Oh. The runtime move to `require_foo` chunk?

Reply:
Yes